errata_tool/erratum.py

```diff
@@ -948,8 +948,10 @@ def commit(self):
             # Errata tool is very slow - don't PUT if it hasn't changed
             allbugs = list(set(self.errata_bugs) | set(
                 self._cve_bugs) | set(self.jira_issues))
-            if sorted(self._original_bugs + self._original_jira_issues) != sorted(allbugs) \
-               or self._update:
+            originalbugs = list(set(self._original_bugs +
+                                    self._original_jira_issues))
+            if (self._update or
+                    list(set(originalbugs).difference(set(allbugs))) == []):
```

Comment:
This actually flips the condition - it would pass of there is no difference between `originalbugs` and `allbugs` while the previous condition was meant to pass if there is a difference

To fix the original condition, we can use `set` instead of `sorted` like so:
```suggestion
            originalbugs = set(self._original_bugs + self._original_jira_issues)
            if (self._update or originalbugs != set(allbugs)):
```
Or just let `allbugs` remain a set to begin with:
```suggestion
            allbugs = let(self.errata_bugs) | set(self._cve_bugs) | set(self.jira_issues)
            originalbugs = set(self._original_bugs + self._original_jira_issues)
            if (self._update or originalbugs != allbugs)):
```